dipy/reconst/multi_voxel.py

```diff
@@ -28,9 +29,13 @@ def new_fit(self, data, mask=None):
 
         # Fit data where mask is True
         fit_array = np.empty(data.shape[:-1], dtype=object)
+        MAXCOUNT = np.prod(data.shape[:-1])
+        bar = tqdm(total=MAXCOUNT, position=0)
         for ijk in ndindex(data.shape[:-1]):
             if mask[ijk]:
                 fit_array[ijk] = single_voxel_fit(self, data[ijk])
+            bar.update()
+        bar.close()
```

Comment:
I would use the total number of voxels in the mask as the total, as I think that gives a more realistic view of the remaining compute to be executed. Something like this: 

```suggestion
        bar = tqdm(total=np.sum(mask), position=0)
        for ijk in ndindex(data.shape[:-1]):
            if mask[ijk]:
                fit_array[ijk] = single_voxel_fit(self, data[ijk])
                bar.update()
        bar.close()
```

Reply:
I'm agree with your suggestion, thank you for that.